### ISSUE TITLE
Win 10 stops In-/Output

### ISSUE THREAD

**Issue by thaloss** [BOINC/boinc#1438]
I've updated from Win7764 to Win10/64 since this update I've got the strange effect that the PC stops Input and Output. This means, the video signal is switched off - Message from Monitor "no input signal" - and the keyboard is not working anymore. I can indicate this because even the [NumLock] and the [CapsLock] which have indication LEDs. the LEDs won't switch.
[Ctrl]+[Alt]+[Del] doesn't work. The PC itself is still working, for example a copy-action which was initialted just before the issue occures, will be finished completely without problem. (I wait for the end of theHD activity)
The PC will run for days without occurance of the isssue if Boinc is not running but If Boinc is launched the issue will occure within the next two hours. 
I thougth of a temprature problem but the sensors shows low temperature (<50° C)

**Comment by SETIguy** [comment]
Do you have a GPU?  Which manufacturer?

**Comment by thaloss** [comment]
I'm using 2x ATI/AMD R7 250X in Crossfire. Manufactured by Saphire.

**Comment by thaloss** [comment]
Due to the question of SETIguy I did the following check.I stopped all projekts which make use of the GPU all other continue working - Result: During 2 days the issue doesn't occur.
i restart just one Projekt which uses the GPU - I decided to use Milkyway@home which has short tasks of a few minutes. Waiting for several tasks to finish without problem. Let the PC tun overnight, next morning the issue has reoccured.
So it is really caused by the GPU usage. Here a bit more details about my System
     CPU - AMD Phenom II Quadcore
     GPU - ATI/AMD R7 250X on 2 boards with Crossfire connection, 1 GB GDDR5 onBoard
     GPU Driver - Catalyst 15.7.1 with display driver 15.20.1062.1004​
     RAM - 2x 4GB
     OS - Windows 10 Pro 64Bit
     BOINC - Version 7.6.9 (x64) without VirtualBox

Something additional, in the calculation settings of BOINC (not projekt) I activated the option to deny GPU usage while the PC is in use (I'm using german translations so I'm not sure about the egnlish description) just activating the option forces the issue to occur !!!
So it seems that the issue is caused by leaving a task which is not finished, like it happens do to task switching after XX minutes, my setting is 60 minutes which explains why it lasts till it occur.
